dj_debexpo/dj_debexpo/wsgi.py

```diff
@@ -0,0 +1,14 @@
+"""
+WSGI config for dj_debexpo project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.7/howto/deployment/wsgi/
+"""
+
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "dj_debexpo.settings")
+
+from django.core.wsgi import get_wsgi_application
+application = get_wsgi_application()
```

Comment:
https://github.com/alex/django-wsgi/issues/1 provides an interesting thought -- sometimes if future Django middleware of ours looks at `HttpResponse.content`, it can cause a side effect of ruining the response by draining the response generator too soon.

They discuss a workaround there, which is -- if you're going to read `response.content`, make sure to stick it back onto the response object, for fear of losing it.

We hopefully won't need that knowledge, but I thought it was worth repeating here.